### PR TITLE
Print status to migrationContext logger

### PR DESCRIPTION
**Pull request by SocalNick**
**NOTE:** This is a simple alternative to #1193 

## A Pull Request should be associated with an Issue.

> We wish to have discussions in Issues. A single issue may be targeted by multiple PRs.
> If you're offering a new feature or fixing anything, we'd like to know beforehand in Issues,
> and potentially we'll be able to point development in a particular direction.

Related issue: https://github.com/github/gh-ost/issues/1026

> Further notes in https://github.com/github/gh-ost/blob/master/.github/CONTRIBUTING.md
> Thank you! We are open to PRs, but please understand if for technical reasons we are unable to accept each and any PR

### Description

Status is currently only sent to stdout. We'd like the status to also be sent to the migrationContext logger.

> In case this PR introduced Go code changes:

- [x] contributed code is using same conventions as original code
- [x] `script/cibuild` returns with no formatting errors, build errors or unit test errors.

### CODE REVIEW

**Review by Svoyu — APPROVED** [review]

**Comment by SocalNick** [comment]
Hi @Svoyu - I see you approved this change, but I can't merge it because none of the workflows have run. It says, "First-time contributors need a maintainer to approve running workflows."

Are you able to approve those to run?

**Review by dm-2 — APPROVED** [review]

**Comment by dm-2** [comment]
> Hi @Svoyu - I see you approved this change, but I can't merge it because none of the workflows have run. It says, "First-time contributors need a maintainer to approve running workflows."
> 
> Are you able to approve those to run?

Hi @SocalNick, I've approved the workflows to run - all looks good! 😄

(P.S. I think @Syoyu is a spam user so I've reported their account)

**Comment by WesleyW** [comment]
Hi there, we're currently using a fork to get this functionality working - are there any plans for a new release soon? We can use the commit hash in the meanwhile if that's not in the works yet. @dm-2

**Pull request merged**
